lib/devise/models/confirmable.rb

```diff
@@ -165,6 +166,12 @@ def skip_reconfirmation!
 
       protected
 
+        # To not require reconfirmation after creating with #save called in a
+        # callback call skip_create_confirmation!
+        def skip_create_confirmation!
```

Comment:
@ArneZsng @ulissesalmeida I feel like we should have better names to express the checks for dealing with the save + after_create scenarios - `skip_create_confirmation` sounds like we are skipping the confirmation on a regular create scenario, regardless of the callback issue.